syntax = "proto2";

package opentxs.proto;
option java_package = "org.opentransactions.proto";
option java_outer_classname = "OTBasketParams";
option optimize_for = LITE_RUNTIME;

import public "BasketItem.proto";
import public "Signature.proto";

message BasketParams
{
    optional uint32 version = 1;
    optional uint64 weight = 2;
    repeated BasketItem item = 3;
}
